core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -237,7 +237,7 @@ public void setCompression(String compressor) {
    * @return Attributes container
    */
   @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1779")
-  public Attributes attributes() {
+  public Attributes getAttributes() {
```

Comment:
Go ahead and have a `@Deprecated` and `@ExperimentalApi` `attributes()` method that calls the new method. That is very easy to do and makes it much easier to migrate.